streams/src/test/resources/log4j.properties

```diff
@@ -18,9 +18,18 @@ log4j.appender.stdout=org.apache.log4j.ConsoleAppender
 log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
 log4j.appender.stdout.layout.ConversionPattern=[%d] %p %m (%c:%L)%n
 
-log4j.logger.kafka=INFO
+log4j.logger.kafka=ERROR
+log4j.logger.state=ERROR
```

Comment:
Not sure what does "log4j.logger.state" mean here? Or did you mean to write "log4j.logger.state.change.logger"?

Reply:
Ah, yeah, I guess there isn't anything else under `log4j.logger.state` besides the `state.change.logger` -- I'll update this